ttnn/module.mk

```diff
@@ -5,3 +5,53 @@
 ttnn/dev_install: python_env/dev
 	echo "Installing editable dev version of ttnn package..."
 	bash -c "source $(PYTHON_ENV)/bin/activate && pip install -e ttnn"
+
+TTNN_LIB = $(LIBDIR)/libttnn.so
```

Comment:
OK so a couple of things

- Do we plan to use any ttnn C++ sources as a C++ library?
- Do we plan to release separate wheels for ttnn in the future?

For some background: right now we only have the python bindings for eager and the metal runtime as shared objects. Everything else is an archive that is meant to be deleted and not used in production.

Reply:
Yes, we want to have two C++ libraries.  One will be without the pybind11 and is intended for folks who do not want to use any C++.  The other is for pybind11 and is intended for using with python.

Yes, I think we should release separate wheels for ttnn.  Any help on getting this setup would be huge.